src/animate/load.ts

```diff
@@ -51,9 +56,10 @@ const EXPECTED_ASSET_VERSION = 2;
  * ```
  * @param scene - Reference to the scene data.
  * @param complete - The callback function when complete.
+ * @param progress - The callback function when progressed.
  * @return instance of PIXI resource loader
  */
-export function load(scene: AnimateAsset, complete?: Complete): void;
```

Comment:
This function signature should remain unchanged - your implementation only handles it in a LoadOptions object (which I agree with doing).

Reply:
Done